x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row_messages_pane.tsx

```diff
@@ -55,5 +55,14 @@ export const ExpandedRowMessagesPane: FC<Props> = ({ analyticsId }) => {
 
   useRefreshAnalyticsList({ onRefresh: getMessagesFactory() });
 
-  return <JobMessages messages={messages} loading={isLoading} error={errorMessage} />;
+  const refreshMessage = useCallback(() => getMessagesFactory(), [analyticsId]);
```

Comment:
In my tests, I'm not seeing `getMessages` being called when I click on the refresh icon in the analytics job table. For the anomaly detection jobs list, it all works as expected.

Reply:
Fixed in https://github.com/elastic/kibana/pull/67750/commits/e458a0aba3e4471b4c2eefcfb6299a7cc3b194dc